lang/csharp/src/apache/ipc/Responder.cs

```diff
@@ -181,14 +181,14 @@ private Protocol Handshake(Decoder input, Encoder output, Transceiver connection
                     WriteResponse(m.Response, response, output);
                 else
                 {
-                    try 
-                    {
-                        WriteError(m.SupportedErrors, error, output);
-                    } 
-                    catch (Exception)
-                    {    
-                        // Presumably no match on the exception, throw the original
-                        throw error;
+                    try 
```

Comment:
Hm, for some reason Github tells that there was some modification, what? It seems unrelated to insecure urls.

Reply:
Strange, When doing a git blame on my machine I only see the copyright url change near the top of the file.

Reply:
Found it: There were trailing spaces on those lines.